ee/localserver/jwk_keys.go

```diff
@@ -0,0 +1,89 @@
+package localserver
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"math/big"
+)
+
+type jwk struct {
+	Type  string `json:"kty"`
+	Curve string `json:"crv"`
+	X     string `json:"x"`
+	Y     string `json:"y"`
+	KeyID string `json:"kid"`
+}
+
+// jwkToECDSAPublicKey converts a JWK JSON string into an ECDSA public key.
+func (jwk jwk) ecdsaPubKey() (*ecdsa.PublicKey, error) {
```

Comment:
Same shadowing thing, plus maybe we want a pointer receiver here? `func (j *jwk)`? Ditto for `x25519PubKey` below.